src/utils/prettify.ts

```diff
@@ -1,24 +1,29 @@
-import path from "path";
 import fs from "fs";
-import shell from "shelljs";
+import fg from "fast-glob";
 import log from "./logging";
+import * as prettier from "prettier";
 
-const prettify = (glob: string) => {
-  const prettierExecutable = path.join(
-    process.cwd(),
-    "node_modules",
-    ".bin",
-    "prettier"
-  );
+const prettify = async (globPattern: string) => {
+  const configFilePath = await prettier.resolveConfigFile();
+  const config = await prettier.resolveConfig(configFilePath || "");
 
-  if (fs.existsSync(prettierExecutable)) {
-    log.info(`Formatting ${glob} files with Prettier`);
-    shell.exec(
-      [prettierExecutable, "--loglevel silent", "--write", `"${glob}"`].join(
-        " "
-      )
-    );
-  }
+  fg.glob(globPattern)
+    .then(files => {
+      files.forEach(async file => {
+        const data = fs.readFileSync(file, "utf-8");
```

Comment:
Async operations inside `forEach` don't work as expected as it the `forEach` function doesn't await the next operation until running the next one. I'm not sure if all things are working correctly here, but if not we can use:

```ts
for (const file of files) {
  await formatFile()
}
```